desktop-exporter/app/components/detail-view/links-panel.tsx

```diff
@@ -0,0 +1,82 @@
+import React from "react";
+
+import {
+  Accordion,
+  AccordionButton,
+  AccordionIcon,
+  AccordionItem,
+  AccordionPanel,
+  Box,
+  List,
+  TabPanel,
+  Text,
+} from "@chakra-ui/react";
+
+import { LinkData } from "../../types/api-types";
+import { SpanField } from "./span-field";
+import { UnderConstructionAlert } from "../alerts/under-construction";
+
+type LinksPanelProps = {
+  links: LinkData[] | undefined;
+};
+
+export function LinksPanel(props: LinksPanelProps) {
+  let { links } = props;
+  if (!links) {
+    return null;
+  }
+
+  // As in a list of links, not the data structure. I'm sorry. Names are hard.
+  let linkList = links.map((link) => {
+    let linkAttributes = Object.entries(link.attributes).map(([key, value]) => (
+      <li key={key + value}>
+        <SpanField
+          fieldName={key}
+          fieldValue={value}
+        />
+      </li>
+    ));
+
+    return (
+      <li key={link.traceID + link.spanID}>
+        <AccordionItem>
+          <AccordionButton>
+            <Box
+              flex="1"
+              textAlign="left"
+            >
+              <Text fontSize="sm">
+                Trace ID: <strong>{link.traceID}</strong>
+              </Text>
+              <Text fontSize="sm">
+                Span ID: <strong>{link.spanID}</strong>
+              </Text>
+            </Box>
+            <AccordionIcon />
+          </AccordionButton>
+          <AccordionPanel>
+            <SpanField
+              fieldName="trace state"
+              fieldValue={link.traceState}
+            />
+            <List>{linkAttributes}</List>
+            <SpanField
+              fieldName="dropped attributes count"
+              fieldValue={link.droppedAttributesCount}
+              hidden={!link.droppedAttributesCount}
+            />
+          </AccordionPanel>
+        </AccordionItem>
+      </li>
+    );
+  });
```

Comment:
Once a `map` gets this large, I recommend breaking it out as a separate component

```jsx
linkListItems = links.map((link) => <LinkItem link={link}/>)
```

Reply:
Done here and for the same issue in the events tab